cobalt/site/docs/gen/cobalt/doc/docker_build.md

```diff
@@ -67,4 +79,4 @@ launching the bash shell, i.e.
 
   `docker-compose run linux-x64x11 /bin/bash`
 
-and try to build cobalt [with the usual `gyp / ninja` flow](../../README.md#building-and-running-the-code).
+and try to build cobalt [with the usual `gn / ninja` flow](../../README.md#building-and-running-the-code).
```

Comment:
😬

Reply:
Hey look, i fixed it. There's quite few other gyp mentions in the docs but i don't need to win them all right away.